github/CheckRun.py

```diff
@@ -169,6 +169,17 @@ def url(self):
         self._completeIfNotSet(self._url)
         return self._url.value
 
+    def rerequest(self):
```

Comment:
```suggestion
    def rerequest(self) -> bool:
```

Reply:
Yes, the `pyi` file has been removed in main branch, so this typing has to go into the `py` file.